webextension/popup.js

```diff
@@ -101,6 +101,31 @@ function getShortCode(languageCode) {
     return languageCode.replace(/-.*/, "");
 }
 
+function suggestionBackgroundStyle(match){
```

Comment:
This should be named `suggestionClass` or `suggestionTypeName`, or similar.
While it is being used for background style now, it might be used for other purposes later, including JS logic and not CSS logic.